docs/UserGuide.md

```diff
@@ -335,7 +335,8 @@ The ignore glob list can be specified through the [standalone config file](#prov
 #### Q: My commit contributions does not appear in the ramp chart (despite appearing in the contribution bar and code panel)?
 **A:** This is probably a case of giving an incorrect author name alias (or github ID) in your [author-config file](#author-configcsv).<br>
 Please refer to [A Note About Git Author Name](#a-note-about-git-author-name) above on how to find out the correct author name you are using, and how to change it.<br>
-Also ensure that you have added all author name aliases that you may be using (if you are using multiple computers or have previously changed your author name).
+Also ensure that you have added all author name aliases that you may be using (if you are using multiple computers or have previously changed your author name).<br>
```

Comment:
Perhaps add the command `git config user.name` as well which helps them find the author name?

Reply:
>Perhaps add the command `git config user.name` as well which helps them find the author name?

I have already referenced to the `A Note About Git Author Name` section of the UserGuide at the FAQ, but seems like it only mentions changing of the author name and not just for finding it. Well add to that section how to find the author name.

Reply:
@eugenepeh I have also updated the current command for setting of git author names for current git repository, as I tested that adding the `--global` flag may not affect the settings for the current git repository.